public/app/features/plugins/admin/components/InstallControls.tsx

```diff
@@ -94,6 +94,11 @@ export const InstallControls = ({ plugin, isInflight, hasUpdate, isInstalled, ha
     );
   }
 
+  if (!hasPermission) {
+    const message = `You need server admin privileges to ${isInstalled ? 'uninstall' : 'install'} this plugin.`;
```

Comment:
Is it possible to leave a bit more space between the text and button?

![image](https://user-images.githubusercontent.com/1173299/128144199-d9b57d4a-0219-477b-a958-20eada587fcd.png)

Reply:
You still see the button? I moved this logic above the button rendering so they don't show any more. 🤔